azure-pipelines.yml

```diff
@@ -21,38 +21,38 @@ pool:
   vmImage: 'Ubuntu 16.04'
```

Comment:
Maybe we should update this to ubuntu-latest to keep consistency with the Github CI? :thinking:

Reply:
I don't see any real benefit of maintaining both azure pipelines and GitHub actions in this repository. I will prefer to remove azure pipelines. If anyone disagrees, please do comment.